plugins/modules/mso_schema_template_anp_epg_useg_attribute.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Sabari Jaganathan (@sajagana) <sajagana@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: mso_schema_template_anp_epg_useg_attribute
+short_description: Manage EPG subnets in schema templates
+description:
+- Manage uSeg Attributes in the schema template EPGs on Cisco ACI Multi-Site.
+author:
+- Sabari Jaganathan (@sajagana)
+options:
+  schema:
+    description:
+    - The name of the Schema.
+    type: str
+  template:
+    description:
+    - The name of the Template.
+    type: str
+  anp:
+    description:
+    - The name of the Application Profile.
+    type: str
+  epg:
+    description:
+    - The name of the EPG.
+    type: str
+  name:
+    description:
+    - The name and display name of the uSeg Attribute.
+    type: str
+    aliases: [ useg ]
+  description:
+    description:
+    - The description of the uSeg Attribute.
+    type: str
+    aliases: [ descr ]
+  attribute_type:
+    description:
+    - The type of the uSeg Attribute.
+    type: str
+    choices: [ vm_name, ip, mac, vmm_domain, vm_operating_system, vm_tag, vm_hypervisor_identifier, dns, vm_datacenter, vm_identifier, vnic_dn ]
+    aliases: [ attr_type ]
+  value:
+    description:
+    - The value the uSeg Attribute.
+    type: str
+  operator:
+    description:
+    - The operator type of the uSeg Attribute.
+    type: str
+    choices: [ equals, contains, starts_with, ends_with ]
+  useg_subnet:
+    description:
+    - The uSeg Subnet only used when the I(attribute_type) is IP.
```

Comment:
- The uSeg Subnet can only be used when the I(attribute_type) is IP.
- Use C(false) to set the custom uSeg Subnet IP address to the uSeg Attribute.
- Use C(true) to set the uSeg Subnet IP address to 0.0.0.0.